internal/kubernetes/dynamic_client.go

```diff
@@ -99,7 +125,7 @@ func ApplyCRSpec(ctx context.Context, restConfig *rest.Config, object *GenericOb
 		}
 
 	} else {
-		numaLogger.Debugf("found existing Resource definition for %s/%s: %+v", object.Namespace, object.Name, resource)
+		numaLogger.Debugf(fmt.Sprintf("found existing Resource definition for %s/%s: %+v", object.Namespace, object.Name, resource))
```

Comment:
why is this needed?

Reply:
the whole thing is a string, right?

Reply:
`Debugf` should resolve the formatting by itself

Reply:
then, you don't need  `%s/%s: %+v` IIUC.

Reply:
revert this back as this is `Debugf` which should be the desired behavior. I was not able to see the complete messages which should be a logger issue then.